packages/patterns/src/patterns/patternMatchers.js

```diff
@@ -1788,25 +1792,52 @@ const makeAwaitArgGuard = argPattern => {
   return result;
 };
 
-const PatternListShape = M.arrayOf(M.pattern());
+// M.rawValue()
+
+// TODO does not need to be a singleton, and would not be if we added a
+// parameter, like a description string.
+/** @type {RawValueGuard} */
+const TheRawValueGuard = harden({
+  klass: 'rawValueGuard',
+});
+
+const RawValueGuardShape = TheRawValueGuard;
+
+export const isRawValueGuard = specimen =>
+  matches(specimen, RawValueGuardShape);
+
+export const assertRawValueGuard = specimen =>
+  mustMatch(specimen, RawValueGuardShape, 'rawValueGuard');
+
+/**
+ * @returns {RawValueGuard}
+ */
+const makeRawValueGuard = () => TheRawValueGuard;
+
+// M.call(...)
+// M.callWhen(...)
```

Comment:
Is the appearance of these two comment lines here a copy-paste error?

Reply:
No, just an artifact of the diff algorithm finding the inserted `M.rawValue()` lines.